libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryWaterfallTests.java

```diff
@@ -249,7 +253,10 @@ public void ensureCancelDialogCalled() {
             .startTest()
             .join();
 
-        Assert.assertEquals(7, counter[0]);
+        Mockito.verify(telemetryClient, Mockito.times(7)).trackEvent(
+            Mockito.anyString(),
+            Mockito.anyMap()
+        );
```

Comment:
Same

Reply:
Same here, this line represents [this one](https://github.com/microsoft/botbuilder-dotnet/blob/main/tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/TelemetryWaterfallTests.cs#L216) from C#.